### PR TITLE
Add support for iojs & nodejs 0.12

### LINKED ISSUE #169: Support for io.js and node 0.12
protagonist-0.19.0 (released Mar 27, 2015) adds support for node 0.12 and io.js. It would great to get dredd to update the dependency.

### PR DESCRIPTION
**Pull request by kuba-kubula**
- add support for io.js and node.js 0.12, closes #169, possibly fixing #175 
- introduces support for MSON requested in #171
- achieved thanks to using [drafter](https://www.npmjs.org/package/drafter), no more protagonist
- using [pitboss-ng](https://www.npmjs.org/package/pitboss-ng) to do cross-platform sandboxing

### CODE REVIEW

**Comment by netmilk** [comment]
it's worth mentioning that solves #169

**Comment by kuba-kubula** [comment]
@netmilk I just added it to the description of the PR, thanks for reminder. And also - it might solve #175 ... maybe.

**Comment by netmilk** [comment]
...and this imho solves request for support of MSON in #171 because Drafter automatically expands MSON Attributes to `body` and `bodySchema` in the AST

File: src/reporters/apiary-reporter.coffee [hunk 1]
```diff
@@ -133,6 +137,19 @@ class ApiaryReporter
         logger.complete "See results in Apiary at: #{reportUrl}"
         callback()
 
+  _interestingAstToSend: (blueprintData = {}) ->
```
Comment (netmilk) [inline src/reporters/apiary-reporter.coffee]: Why `interesting`? Please name it properly. I can imagine something like `truncated` or `limited`.

**Pull request merged**